test/systemtests/docker_test.go

```diff
@@ -246,6 +246,122 @@ func (d *docker) startListener(c *container, port int, protocol string) error {
 	return err
 }
 
+func (d *docker) startIperfServer(c *container) error {
+
+	_, err := d.execBG(c, fmt.Sprintf("iperf -s"))
+	return err
+}
+
+func (d *docker) startIperfClient(c *container, ip, limit string, isErr bool) error {
+
+	var (
+		bwLimit int64
+		bwInt64 int64
+	)
+	bw, err := d.exec(c, fmt.Sprintf("iperf -c %s ", ip))
+	logrus.Infof("starting iperf client on conatiner:%s for server ip: %s", c, ip)
+	if err != nil {
+		logrus.Errorf("Error starting the iperf client")
+	} else {
+		if strings.Contains(bw, "bits/sec") {
+			bwString := strings.Split(bw, "Bytes ")
+			bwInt64, err = BwConvertInt64(bwString[1])
+			if err != nil {
+				return err
+			}
+			if limit != "" {
+				bwLimit, err = BwConvertInt64(limit)
+				if err != nil {
+					return err
+				}
+				bwLimit = bwLimit + (bwLimit / 10)
+				if bwLimit > bwInt64 {
+					//	logrus.Infof("Obtained bandwidth :%dkbits is less than the limit:%dkbits", bwInt64, bwLimit)
+					logrus.Infof("Obtained bandwidth :%s is less than the limit:%s", strings.TrimSpace(bwString[1]), limit)
+				} else if bwLimit < bwInt64 {
+					if isErr {
+						logrus.Errorf("Obtained Bandwidth:%s is more than the limit: %s", strings.TrimSpace(bwString[1]), limit)
+					} else {
+						logrus.Errorf("Obtained bandwidth:%s is more than the limit %s", bwString[1], limit)
+						return errors.New("Applied bandwidth is more than bandwidth rate!")
+					}
+				} else {
+					logrus.Errorf("Bandwidth rate :%s not applied", limit)
+					return errors.New("Bandwidth rate is not applied")
+				}
+			} else {
+				logrus.Infof("Obtained bandwidth:%s", bwString[1])
+			}
+		} else {
+			logrus.Errorf("Bandwidth string invalid:%s", bw)
+		}
+	}
+	return err
+}
+
+func BwConvertInt64(bandwidth string) (int64, error) {
+	var (
+		rate      int64
+		kiloBytes int64
+		magaBytes int64
```

Comment:
possible to use constants here, check some implementations like in here https://github.com/cloudfoundry/bytefmt/blob/master/bytes.go